src/fluree/server/consensus/standalone.clj

```diff
@@ -0,0 +1,135 @@
+      (broadcast-new-ledger! subscriptions watcher (assoc commmit-result :tx-id tx-id
+                                                          :ledger-id ledger-id
+                                                          :t 1)))))
+
+(defn broadcast-new-commit!
+  "Responsible for producing the event broadcast to connected peers."
+  [subscriptions watcher {:keys [ledger-id tx-id server commit-file-meta] :as commit-result}]
+  (log/info "New transaction completed for" ledger-id "tx-id: " tx-id "by server:" server)
+  (watcher/deliver-watch watcher tx-id commit-result)
+  (subscriptions/send-message-to-all subscriptions "new-commit" ledger-id (:json commit-file-meta))
+  :success)
+
+(defn transact!
+  [conn subscriptions watcher {:keys [ledger-id tx-id txn opts] :as params}]
+  (go-try
+    (let [start-time    (System/currentTimeMillis)
+          _             (log/debug "Starting transaction processing for ledger:" ledger-id
+                                   "with tx-id" tx-id ". Transaction sat in queue for"
+                                   (- start-time (:instant params)) "milliseconds.")
+          ledger        (if (deref! (fluree/exists? conn ledger-id))
+                          (deref! (fluree/load conn ledger-id))
+                          (throw (ex-info "Ledger does not exist" {:ledger ledger-id})))
+          staged-db     (-> ledger
+                            fluree/db
+                            (fluree/stage txn opts)
+                            deref!)
+          commit-result (deref!
+                         (fluree/commit! ledger staged-db {:file-data? true}))
+          broadcast-msg (msg-format/new-commit nil params commit-result)]
+      (broadcast-new-commit! subscriptions watcher broadcast-msg))))
+
+(defn process-event
+  [conn subscriptions watcher event]
+  (go
+    (try
+      (let [[event-type event-msg] event
+
+            result (<? (case event-type
+                         :ledger-create (create-ledger! conn subscriptions watcher event-msg)
+                         :tx-queue      (transact! conn subscriptions watcher event-msg)))]
+        result)
+      (catch Exception e
+        (log/error "Unexpected event message - expected two-tuple of [event-type event-data], "
+                   "and of a supported event type. Received:" event e)))))
+
+(defn new-tx-queue
+  [conn subscriptions watcher]
+  (let [tx-queue (async/chan 512)]
+    (go
+      (loop [i 0]
+        (let [timeout-ch (async/timeout 5000)
+              [event ch] (async/alts! [tx-queue timeout-ch])]
+          (cond
+            (= timeout-ch ch)
+            (do
+              (log/trace "No new transactions in 5 second."
+                         "Processed" i "total transactions.")
+              (recur i))
+
+            (nil? event)
+            (do
+              (log/warn "Closing local transaction queue was closed. No new transactions will be processed.")
```

Comment:
My typo.. but should be:
"Local transaction queue was closed. No new transactions will be processed."